gc/stats/AllocationStats.hpp

```diff
@@ -59,14 +60,22 @@ class MM_AllocationStats : public MM_Base
 
 #if defined(OMR_GC_THREAD_LOCAL_HEAP)
 	uintptr_t tlhBytesAllocated() { return _tlhAllocatedFresh - _tlhDiscardedBytes; }
+	uintptr_t tlhBytesAllocatedUsed() { return _tlhAllocatedUsed; }
 	uintptr_t nontlhBytesAllocated() { return _allocationBytes; }
 #endif
 
-	uintptr_t bytesAllocated(){
+	/* return bytesAllocated includes new refreshed TLH, if used == false(default)
+	 * return bytesAllocated (but does not include new refreshed TLH), if used == true. 
+	 */
+	uintptr_t bytesAllocated(bool used = false) {
```

Comment:
The flag should more resemble to your comment (and invert the logic, if needed). Something like include(Just?)RefreshedTLH ?